Code/GraphMol/FileParsers/MolFromScsr.cpp

```diff
@@ -0,0 +1,494 @@
+//
+//  Copyright (C) 2002-2024 Tad Hurst and other RDKit contributors
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "MolSGroupParsing.h"
```

Comment:
The windows build failure might be due to not including  `FileParsers.h` for the `MolFromScsr` definition

Reply:
MolFromScsr.cpp has been removed and the code that was in it is now in ScsrMolFileParser.cpp.   That file contains a reference to #include "FileParserUtils.h", which includes "FileParsers.h".